### ISSUE TITLE
Multi Level Select

### ISSUE THREAD

**Issue by marvink** [rah003/scripted-select#3]
It would be realy nice if there would be a select field which als shows multiple level of a jcr structure. This could be a much better option then using multivalue fields to choose for example a lot of categories inside the category app.

**Comment by rah003** [comment]
True, but that would be slightly different feature than what this module is providing right? Are you using the module or you found it just when looking for what you describe above? If using, could you pls share with which version of Magnolia? (just so i know where it still works and what would be the minimal version that needs to be supported if i decide to tackle your suggestion). thx
